lambdas/process-journey-event/src/main/resources/statemachine/journey-maps/repeat-fraud-check.yaml

```diff
@@ -226,6 +226,9 @@ states:
       enhanced-verification:
         targetJourney: FAILED
         targetState: FAILED_CONFIRM_DETAILS
+      fraud-fail-with-no-ci:
+        targetJourney: FAILED
+        targetState: FAILED
```

Comment:
This is a slightly interesting one - my assumption is that we _should_ allow them to get a repeat M1C if Experian isn't usable.

(Not relevant to international address, but will be for fraud resilience)